libs/VideoKit/include/VideoKit.h

```diff
@@ -4,24 +4,46 @@
 
 #pragma once
 
+#include "rtos/Thread.h"
+
+#include "interface/drivers/EventFlags.h"
+#include "interface/drivers/LCD.hpp"
+#include "interface/drivers/Video.h"
 #include "interface/libs/VideoKit.h"
 
 namespace leka {
 
 class VideoKit : public interface::VideoKit
 {
   public:
-	explicit VideoKit() = default;
+	explicit VideoKit(interface::EventFlags &event_flags, interface::LCD &lcd, interface::Video &video)
+		: _event_flags(event_flags), _lcd(lcd), _video {video}
+	{
+	}
```

Comment:
j'aime bien avoir un commentaire pour dire qu'on fait volontairement rien ici et que c'est pas un oubli

```suggestion
	{
		// nothing to do
	}
```